src/shared/ui/components/BaseTable.vue

```diff
@@ -81,6 +83,13 @@ interface Props {
   reversed?: boolean
 }
 
+const emit = defineEmits<{
+  'click:row': [data: T]
+}>();
+
+const instance = getCurrentInstance();
+const hasClickRowHandler = computed(() => !!instance?.vnode?.props?.['onClick:row']);
```

Comment:
If it isn't _really_ necessary to access raw VNodes and other Vue internals, better not to.

The same could be achieved by `useAttrs()`, or by simply providing an explicit boolean prop like `row-pointer`.